frontend/web/components/FlagOwnerGroups.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react'
+import data from 'common/data/base/_data'
+import UserSelect from './UserSelect'
+import ConfigProvider from 'common/providers/ConfigProvider'
+import Icon from './Icon'
+import { close } from 'ionicons/icons'
+import { IonIcon } from '@ionic/react'
+import GroupSelect from './GroupSelect'
+import { getProjectFlag } from 'common/services/useProjectFlag'
+import { getStore } from 'common/store'
+
+class TheComponent extends Component {
+  state = {}
+  componentDidMount() {
+    this.getData()
+  }
+
+  getData = () => {
+    getProjectFlag(getStore(), {
+      id: this.props.id,
+      project: this.props.projectId,
+    }).then((res) => {
+      const owners = (res.data.group_owners || []).map((v) => v.id)
+      this.setState({ owners })
+    })
+  }
+
+  addOwner = (id) => {
+    this.setState({ owners: (this.state.owners || []).concat(id) })
+    data.post(
+      `${Project.api}projects/${this.props.projectId}/features/${this.props.id}/add-group-owners/`,
+      {
+        group_ids: [id],
+      },
```

Comment:
I was anticipating selecting multiple group owners at once, but I guess the UI only needed one at a time.

Reply:
Yeah, same as standard users for now, can adjust both together if the UI changes